style.css

```diff
@@ -607,10 +615,44 @@ label {
   }
 
   .about_description {
-    width: 60vw;
+    width: 68vw;
+  }
+
+  .about_container {
+    background-image:
+      url(./assets/background/illustration-about-me-2-desktop.svg),
+      url(./assets/background/illustration\ 1\ about\ me\ desktop.svg);
+    background-size: 15%, 85%;
+  }
+
+  .about__list {
+    display: flex;
+    flex-basis: 28, 3333%;
+    flex-wrap: wrap;
+    justify-content: center;
   }
 
   .skills {
     grid-area: 2/1/3/3;
   }
+
+  .contact__form ul {
+    grid-template-columns: 1fr 1fr;
+    grid-template-areas:
+      "name lastname"
+      "email email"
+      "message message"
+      "button__form button__form";
+    gap: 24px;
```

Comment:
- [ ] I would kindly sugget you to stick to the figma design as much as posible ,Kindly consider your contact details shouldnt be at the top of the contact form but to be align at left side just exact like figma design to make your website look professional,also this will show the recruilter in future how much you stick to instruction and given guildline .your contact section details design is not the same to the figm design you chose mthe texts is not at the top of the contact form but beside the form left side see the screenshot below for comparism

- [ ] your design
![Screenshot 2023-06-10 031644](https://github.com/mateoburvi/Portfolio/assets/92458236/de382c05-4e21-4f2f-9cf6-0ff39dbf6b92)

- [ ] figma design
![Screenshot 2023-06-10 030236](https://github.com/mateoburvi/Portfolio/assets/92458236/9d4830c6-c486-44d7-b70d-207a0431bb40)